Tribler/Core/Upgrade/upgrade.py

```diff
@@ -47,6 +47,7 @@ def run(self):
 
                 self.upgrade_to_tribler7()
                 self.session.config = convert_config_to_tribler71()
+                self.session.config.set_upgrader_enabled(False)
```

Comment:
Doesn't it make sense to disable the upgrader after the version is just upgraded? Where do you suggest to move it?